src/StepperFirmata2.cpp

```diff
@@ -87,7 +87,7 @@ boolean StepperFirmata2::handleSysex(byte command, byte argc, byte *argv)
   if (command == ACCELSTEPPER_DATA) {
 
     byte stepCommand, deviceNum, interface, wireCount, stepType;
-    byte stepOrMotorPin1, directionOrMotorPin2, motorPin3, motorPin4, enablePin, invertPins = 0;
+    byte stepOrMotorPin1, directionOrMotorPin2, ms1OrMotorPin3, ms2OrMotorPin4, enablePin, invertPins = 0;
```

Comment:
Also, Big Easy Driver has an MS3 pin so that just adds another variation. I'm further convinced that J5 and other dependents of firmata.js are the right place for this.